### PR TITLE
Feature - AdminBook Index [Search]

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

**Comment by gittiggit** [comment]
OK

**Comment by mdmahmudurrahman** [comment]
OK

File: app/models/book.rb [hunk 1]
```diff
@@ -29,27 +32,58 @@ class Book < ActiveRecord::Base
...
+      end
     end
   end
 
+  private
+  # class << self
+  #   def ransackable_scopes auth_object = nil
+  #     [:filter_book]
+  #   end
+  # end
```
Comment (zenith19) [inline app/models/book.rb]: ?

**Comment by sujoydatta** [comment]
ok

**Pull request merged**